data/brands/shop/supermarket.json

```diff
@@ -30,6 +30,17 @@
     }
   },
   "items": [
+    {
+      "displayName": "Reliance Smart Super Store",
+      "locationSet": {"include": ["in"]},
+      "matchNames": ["Reliance Smart","Reliance Mall","Jio Mall","Jio Supermarket"],
+      "tags": {
+        "brand": "Reliance Smart Super Store",
+        "brand:wikidata": "Q7311028",
```

Comment:
Probably should have a separate brand item instead of just putting everything under Reliance Retail.

Reply:
A Wikipedia page only exists for Reliance Retail where all these are mentioned as sub brands.

Reply:
```suggestion
        "brand:wikidata": "Q123565819",
```